src/libs/Input/include/Input.hpp

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <functional>
+#include <variant>
+
+namespace storm
+{
+// TODO: Make own type for keycodes and use it everywhere (reuse SDL scancodes?)
+//! Keyboard key, using virtual key codes (VK_* frim WinUser.h) for now
+using KeyboardKey = unsigned int;
+
+//! Mouse relative position
+// FIXME: Better type for that?
+using MousePos = std::pair<int, int>;
```

Comment:
Using a simple struct should do the trick?
```suggestion
struct MousePos {
    int x{};
    int y{};
};
```